Framework/Algorithms/src/CompareWorkspaces.cpp

```diff
@@ -1081,7 +1109,7 @@ void CompareWorkspaces::doPeaksComparison(PeaksWorkspace_sptr tws1,
       std::string name = col->name();
       double s1 = 0.0;
       double s2 = 0.0;
-      if (name == "runnumber") {
+      if (name == "RunNumber") {
```

Comment:
So this didn't work correctly before for PeaksWorkspace :open_mouth: 

A quick test show you did fix this for PeaksWorkspace, this show matched on master but mismatch on this branch :+1: 
```python
ws = CreateSampleWorkspace()
peaks = CreatePeaksWorkspace(InstrumentWorkspace='ws')
peaks2 = CreatePeaksWorkspace(InstrumentWorkspace='ws')
peaks.getPeak(0).setRunNumber(123)
CompareWorkspaces(peaks, peaks2)
```

It seems some of the other column names are also incorrect for PeakWorkspace

`detid` -> `DetID`
`tof` -> `TOF`
`row` -> `Row`
`col` -> `Col`

So I'm guessing this means there is no real testing for CompareWorkspaces with PeakWorkspace :facepalm:

Reply:
That seems to be the case. Also, I have updated the column names as suggested.